build.gradle

```diff
@@ -14,6 +14,22 @@ apply plugin: 'nebula-aggregate-javadocs'
 apply plugin: 'org.ajoberstar.git-publish'
 apply plugin: 'net.researchgate.release'
 
+jar {
```

Comment:
This needs to be moved down to be inside the `subprojects` block below to be applied to all the subprojects.

The `versioning.info.commit` setting requires the `net.nemerosa.versioning` plugin:
* Add `classpath 'net.nemerosa:versioning:2.7.1'` to the `buildscript` > `dependencies` block at the top of the file.
* Add `apply plugin: 'net.nemerosa.versioning'` just below that block